### DISCUSSION TITLE
关于末流211和双非的抉择问题

### DISCUSSION THREAD

**Discussion by KokuraSona** [CollegesChat/university-information#45] [Q&A] [answered]
如题，鄙人分数比较尴尬，加之想报考计算机类专业，在保证大概率被计算机类专业录取的前提下就只有末流211及以下的学校可选了。关于未流211目前关注了两所（东北林业大学、东北农业大学）因为我就是北方人所以环境适应问题不考虑，仅想知道对于计算机专业，这种末流211和双非的好专业到底哪个更好，希望能得到一些建议，十分感谢🙏

**Comment by jzl2163** [comment, marked as answer]
非计算机专业的也想来掺和一下（
选211的话，如果考研，很有帮助。
选双非，可能就业会好一点。
看你以后想不想深造了。

**Reply by KokuraSona** [reply]
不应该是211更好就业吗（
之前也看过一些学长的分享，有提到部分招聘直接筛掉双非的情况。

**Reply by jzl2163** [reply]
计算机行业首先会看行业强校的吧。
筛简历确实也是一个问题，其实主要看公司怎么决定招聘的了..

**Reply by 794TD** [reply]
能力大概率大于院校，真实案例是北京服装学院计算机类的本科毕业进外企

**Reply by zhaoniubi** [reply]
能力在哪都能培养的主要是人脉然后如果你这个确定在招聘过程中不会受到学历歧视或者受到了学历歧视是由于能力而不是由于学历这个误差小于一定范围可以考虑双非

**Comment by AcCoA** [comment]
双非学校出来冒泡。个人建议按专业排名选，如果专业好未必在乎985和211

**Comment by LordHumphrey** [comment]
1. 东北林业大学、东北农业大学这两所院校计算机实力不用说，比起某些双非还是差一点，但是每年都有去BAT这种一线大厂的，计算机这一行，对于找工作来说，课上教的基本都没啥用，主要看自己学。
2. 你现在如果没考虑好读不读研究生，建议你末流211，末流211计算机保研率还是12%左右，这是双非很难达到的，在这保研的学生当中，去c9的大概1/3左右，所以211给你带来的大环境还是比较好的。
3. 东北林业大学的保研率、教育部经费和学科竞赛获奖情况大概在全国前100名院校中60-80名这样，这三项指标比院校排名高多了，所以 这个地方对得起你这个分。
4. 缺点就是住宿条件，大环境的确不如南方院校，毕竟还是8人寝室。

**Reply by KokuraSona** [reply]
非常感谢，很有帮助🙏

**Comment by palemoons** [comment]
个人的观点是平台更重要。

- 首先211院校能给你更多资源，你认识的同学、老师将来都是你重要的人脉关系。以及一些大厂也更乐意去这些学校招生。
- 其次进了学校之后未必只能在一个专业上吊死，可以试着转专业，当然计算机这种热门专业必然难转。
- 最后对于计算机这种职业并非要在这个专业里才能学，事实上学校里的课程只能给你打个基础，而网上网课资源这么多，文档也非常齐全，完全可以自学。我身边认识的非CS专业最后进厂的也不少。
希望能有所参考。

**Reply by KokuraSona** [reply]
感谢🙏

**Comment by OPLIKATO** [comment]
我是微电子专业的，偏计算机方向的，现在已经毕业了。下面给你参考一下：
1、我同意楼上的观点，计算机专业靠自学，或者自己找导师做项目和竞赛，这对你找工作帮助很大。计算机专业找工作看个人能力的比重大一些。顺便提一句，考虑来微电子专业吗，分数应该没cs那么高。这样你来一个和计算机很接近的专业，可能还能去个好一点的211。微电子的方向可以偏向计算机也可以偏向半导体制造，毕业了想转cs也很轻松。我们专业做cs的也有不少人。
2、关于资源，211里的资源确实会更好。这些资源包括：老师的学历，学术水平，老师手里的项目质量好坏，实验室设备的水平，周围同学的学习氛围（这个也是比较重要的，不要高估自己的定力），校招去的企业水平，保研率（985，211，双非之间差距很大）
3、上面是对于学校整体水平的比较。具体的专业强弱你要自己查一下学校官网。你可以比较一下你心仪的几个学校，计算机专业年轻老师的学历和经历。有的双非可能计算机是强势学科，教师资源很好，那还是建议你去这样的强计算机的双非。
4、如果你和老师混的熟，老师可能会把他自己的导师或者资源介绍给你，那么这种情况下，教师的背景就很重要了。
5、再一个转专业的问题，转专业名额有限，专业不是那么好转的，特别是计算机这种热门方向不太好进。
6、211保研考研出国都比双非优势更大。但是注意，如果以后是在国内发展，有些双非能碾压很多211，比如深圳大学，上海科技大学，南方科技大学之类的。
7、位置上，电子类专业建议你去南方，好找工作。举个例子，对于我的专业，集成电路的大公司都在南方，在南方上学校招的机会会大一点。

**Reply by KokuraSona** [reply]
非常感谢🙏

**Comment by 751K** [comment]
电子信息工程不请自来，坐标北京某211
我先推211一波
1.保研啊，虽然你现在肯定对要不要考研保研不清楚，但是211可以给你一个更轻松保研的底气，你看过他们考研的就知道了，看完绝对不会想要去考研的、、属实太累、、、
2.转专业虽然难转，但是还是有机会，以我校为例，信工院一直是最热门的，每届都能转入不少人，虽热竞争有些激烈，但是实际看下来还好（？）至少还是存在不小可能的，当然你大一成绩要好or高中有信息竞赛这些

**Comment by derrickzhuyz** [comment]
如果要读研的话211客观上有优势，保研率高（虽然浙工大这样的bug双非保研率也有10%），出国也更大概率在list内。如果有读研的打算的话优先选保研率高一点的学校，进去卷到推免名额的可能性更大点。直接就业的话还是要看专业认可度，末流211认可度挺难比得上南邮重邮这样的学校的。

**Comment by LordHumphrey** [comment]
楼主最后去哪里了？

**Comment by Magist0ne** [comment]
2024年了，现在是能211就211，双非找工作人家都不一定能看简历，直接就筛掉了

**Comment by sdfg12-alt** [comment]
我的建议是选211，毕竟是个入场卷
